docs/pages/Builder.md

```diff
@@ -83,6 +84,55 @@ Expand the block below to view a diagram illustrating the minification process a
 ??? info "Minification Activity Diagram"
     ![minify Task Activity](../images/UI5_Tooling/Task_Minify.svg){ loading=lazy }
 
+
+### Generation of supported locales
+
+The `enhanceManifest` task fills the `supportedLocales` property in the `manifest.json` of a UI5 library/application automatically with the available locales determined by the existence of the respective `.properties` translation file.
```

Comment:
The minimum "_version" should be mentioned

Reply:
It should also be mentioned that this behavior won't apply for files outside of the namespace of the manifest.json.

Reply:
done